packages/cashier/src/stores/iframe-store.ts

```diff
@@ -79,7 +80,7 @@ export default class IframeStore {
         }
     }
 
-    setIframeUrl(url?: string): void {
+    setIframeUrl(url?: Cashier): void {
```

Comment:
```suggestion
    setIframeUrl(url?: string): void {
```
@hamid-deriv I guess `string` is the correct type.